redbadger/continous-compliance
# Setup Typescript project

On this PR we are setting the dev tools to create a GitHub Action on JavaScript

## How to test:
- On this branch install dependencies running `yarn`
- To compile TypeScript to JavaScript run `yarn build`
- Check `/dist` folder

![](https://media.giphy.com/media/24FCjWbQyY4M4xJUeK/giphy.gif)
